### ISSUE TITLE
Horizontal bar not visible when image/SVG larger than window

### ISSUE THREAD

**Issue by BackOrder** [Shoes3/shoes3#163]
![image](https://cloud.githubusercontent.com/assets/8844578/11915202/ec28dc76-a669-11e5-9de7-03acce5f4ff1.png)

```
Shoes.app(width: 256, height: 256) {
  image("#{DIR}/static/splash.png")
}
```

**Comment by ccoupe** [comment]
This would require horizontal scrolling for all things since it's implemented in canvas and it  would change how Shoes lays out widgets. It runs counter to the Shoes design of a fixed with, infinite height app windows. In short, a complete rewrite of the layout engine and modifications to all  widgets. One hell of a lot of work. 

I know you think I'm exaggerating or I'm being argumentative because I'm stubborn. Feel free to study the C code and fix it. The only question for me is whether to assign this issue to enhancement or wish list.  3.3. is not likely.

**Comment by BackOrder** [comment]
You may be absolutely right but it is not an enhancement, it is an actual bug. Removing 3.3.0 tag is fine in the light of your feedback. There is no urgency on this one at the moment. So we can keep it, document it, look at it later on. No problem.

**Comment by BackOrder** [comment]
(Small update) This bug is still present in Shoes 3.3.2.

**Comment by ccoupe** [comment]
No, it is not a bug - I don't remember Shoes **ever** putting horizontal scroll bars on stacks or flows. The vertical scroll bar is for the slot, not the image.

**Comment by passenger94** [comment]
This is by design, an assumed choice by the creator himself, Manual is crystal clear on this !!!

**Comment by BackOrder** [comment]
@passanger94 would you please point exactly in the manual where it is crystal clear? This would make a whole lot difference as to what we should do with this issue.

**Comment by ccoupe** [comment]
It doesn't require  manual confirmation (a fallible gotcha) . Slots don't get horizontal scrolling.  They just don't. Never did. Edit_box (a native) might  or might not but Shoes doesn't know about that.

**Comment by ccoupe** [comment]
Furthermore , svg resizes to fit the slot. Try it. You can also scale a  png to fit into your predetermined size. 
```ruby
Shoes.app(width: 256, height: 256) {
  image "#{DIR}/static/splash.png", width: 256, height: 256
}
```
![tiny-walkabout](https://cloud.githubusercontent.com/assets/222691/22395304/987efd9a-e4f6-11e6-8b69-ff04fbff270e.png)

**Comment by passenger94** [comment]
See beginning of the Slot section.
There's some other info elsewhere, i can't find it atm, maybe Nobody Knows Shoes or the Wiki ...
This is a central design of Shoes, (think about the Flow's behavior @cecil is talking about)
